Source/SteveSloth/Public/MyEnemyBaseClass.h

```diff
@@ -56,16 +56,16 @@ class STEVESLOTH_API AMyEnemyBaseClass : public ACharacter
 	float RangedAttackRange;
 	float MaxConfusionRange;
 
-	bool bIsFrozen;
-	bool bIsCurrentlyFrozen;
-	bool bIsConfused;
-	bool bIsCurrentlyConfused;
 	bool bIsDead;
 	bool bIsIdle;
+	bool bIsFrozen;
 	bool bIsChasing;
+	bool bIsConfused;
 	bool bIsPatroling;
 	bool bIsAttackingMelee;
+	bool bIsCurrentlyFrozen;
 	bool bIsAttackingRanged;
+	bool bIsCurrentlyConfused;
```

Comment:
I understand what you were doing here.. but you were suppose to be working on the player not organizing other code.. things like this mess up code.. stick to what you were doing..